### PR TITLE
75462 - Webhook Authentication Rbac Jwt Token

### PR DESCRIPTION
**Pull request by Shirin14926**
Code changes-Authentication and Configuration

### CODE REVIEW

**Review by jrippington — APPROVED** [review]
Looks good!

File: tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/WebhookEndpoint.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            var request = new RestRequest("/webhook/newenccontentpublishedeventoptions");
+            var response = await client.OptionsAsync(request);
+            return response;
+        }
+
+        
+        public async Task<RestResponse> PostWebhookResponseAsync(String filePath)
+        {
+            string requestBody;
+            using (StreamReader r = new StreamReader(filePath))
```
Comment (harshal11869) [inline tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/WebhookEndpoint.cs:28]:
```suggestion
            using (StreamReader streamReader = new StreamReader(filePath))
```

**Review by harshal11869 — CHANGES_REQUESTED** [review]

File: tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/WebhookEndpoint.cs (hunk 1 above)
Comment (Shirin14926) [reply tests/UKHO.ERPFacade.API.FunctionalTests/Helpers/WebhookEndpoint.cs]: done. thanks

**Review by harshal11869 — APPROVED** [review]
Looks Good. @Shirin14926 Thanks for fixing the review comments.

**Review by jrippington — APPROVED** [review]
Looks good, thanks!

**Review by JiviteshT — APPROVED** [review]
Approved to progress merge

**Pull request merged**